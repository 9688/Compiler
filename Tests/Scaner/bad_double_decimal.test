e1
1e
1.e
.1e
1.1e*1
1.1p1