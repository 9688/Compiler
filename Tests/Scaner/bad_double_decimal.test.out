IDENTIFICATOR 1 1 e1
Ошибка в строке 2 позиции 2: требуется экспоненциальное значение
Ошибка в строке 3 позиции 3: требуется экспоненциальное значение
Ошибка в строке 4 позиции 3: требуется экспоненциальное значение
Ошибка в строке 5 позиции 4: требуется экспоненциальное значение
OPERATOR 5 5 *
DOUBLE 5 6 1
Ошибка в строке 6 позиции 3: необходимо "E" для основания "10"
IDENTIFICATOR 6 4 p1
EOF 6 6 
